scripts/install-kusion.sh

```diff
@@ -22,7 +22,7 @@ for arg in "$@"; do
     fi
 
     # base64 encode
```

Comment:
The new implementation using `cat` and `tr` is less efficient than the original `base64 -w 0`. Consider reverting to the original approach as it's more direct and performs better by avoiding unnecessary pipes.